docs-md/operate-and-deploy/installation/installing.md

```diff
@@ -3,144 +3,222 @@ layout: page
+  ksqlDB Server image
+- [ksqldb-cli](https://hub.docker.com/r/confluentinc/ksqldb-cli/):
+  ksqlDB command-line interface (CLI) image
+- [cp-zookeeper](https://hub.docker.com/r/confluentinc/cp-zookeeper):
+  {{ site.zk }} image (Community Version)
+- [cp-schema-registry](https://hub.docker.com/r/confluentinc/cp-schema-registry):
+  {{ site.sr }} image (Community Version)
+- [cp-kafka](https://hub.docker.com/r/confluentinc/cp-kafka):
+  {{ site.aktm }} image (Community Version)
 
-Start the ksqlDB Server
------------------------
+Install ksqlDB and {{ site.aktm }} by starting a
+[Docker Compose](https://docs.docker.com/compose/) stack that runs containers
+based on these images.
 
-The ksqlDB servers are run separately from the ksqlDB CLI client and {{ site.ak }}
-brokers. You can deploy servers on remote machines, VMs, or containers,
-and the CLI connects to these remote servers.
+The following sections show how to install Docker and use the docker-compose
+tool to download and run the ksqlDB and related images.
 
-You can add or remove servers from the same resource pool during live
-operations, to scale query processing. You can use different resource pools
-to support workload isolation. For example, you could deploy separate pools
-for production and for testing.
+Install Docker
+--------------
 
-You can only connect to one ksqlDB server at a time. The ksqlDB CLI does not
-support automatic failover to another ksqlDB Server.
+Install the Docker distribution that's compatible with your operating system.
 
-![image](../../img/client-server.png)
+!!! important
+    For macOS and Windows, Docker runs in a virtual machine, and you must
+    allocate at least 8 GB of RAM for the Docker VM to run the {{ site.ak }}
+    stack. The default is 2 GB.
+
+- For macOS, use
+  [Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/install/).
+  Change the **Memory** setting on the
+  [Resources](https://docs.docker.com/docker-for-mac/#resources) page to 8 GB.
+- For Windows, use
+  [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/).
+  Change the **Memory** setting on the
+  [Advanced](https://docs.docker.com/docker-for-windows/#advanced) settings
+  page to 8 GB.
+- For Linux, follow the [instructions](https://docs.docker.com/install/)
+  for your Linux distribution. No memory change is necessary, because Docker
+  runs natively and not in a VM.
+
+Create a stack file to define your ksqlDB application 
+-----------------------------------------------------
+
+When you've decided on the services that you want in the stack, you define a
+[Compose file, or "stack" file](https://docs.docker.com/compose/compose-file/),
+which is a YAML file, to configure your ksqlDB application's services. The 
+stack file is frequently named `docker-compose.yml`. 
+
+To start your ksqlDB application, you use the
+[docker-compose CLI](https://docs.docker.com/compose/) to
```

Comment:
...run the stack for the application. Run `docker-compose up` to start the
application and...